src/components/DetailDrawer/index.tsx

```diff
@@ -201,7 +195,7 @@ function DetailDrawer(props: DetailDrawerProps) {
   const drawerIsOpen = Boolean(node);
```

Comment:
its used on lines 300 & 305. the swap from `if(Boolean(node))` to `if (node)` is because of a typescript quirk https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgLIE8DyAjAVhBMZAbwChkLkALCAG1oHsAuZAZzClAHMBuUgX1KlaEImCrdkAXmQgArvWRxWaLHgJEAPrIW1k2uSAAmEGKAhG+pYDGQAKcdwCUJcpQQMQrBiIB0jLgcJEC5fGnoGJwEhG3sAIQYfCDgQIOcXMkpkDy8k-wZAxxCwukYowSA